salt/modules/win_pkg.py

```diff
@@ -29,6 +29,7 @@
 except ImportError:
     import msgpack_pure as msgpack
 # pylint: enable=import-error
+import shlex
```

Comment:
This needs to be `from salt.ext.six.moves import shlex_quote as _cmd_quote`, or similar.